<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chooser</class>
 <widget class="QWidget" name="Chooser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>100</width>
    <height>154</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="ChooserButton" name="pushButtonAnd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>38</y>
     <width>100</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ChooserButton" name="pushButtonOr">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>77</y>
     <width>100</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ChooserButton" name="pushButtonXor">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>116</y>
     <width>100</width>
     <height>38</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="ChooserButton" name="pushButtonNot">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>39</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="checkable">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChooserButton</class>
   <extends>QPushButton</extends>
   <header location="global">chooserbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
